### PR TITLE
examples: Version updates and cleanups

### PR DESCRIPTION
**Pull request by phlax**
- Bump skywalking and resolve version issues
- Bump container/go versions for grpc
- Bump container version for zstd
- Run all verify jobs together (as disk space is less of an issue)
- Use shared/echo container for lua
- Use shared/echo container for wasm
   
Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by phlax** [comment]
@adisuissa this one would be good to land

im trying to clean up the examples before release - with the hope of reducing flakes on the release branch

File: ci/do_ci.sh [hunk 1]
```diff
@@ -505,7 +505,7 @@ elif [[ "$CI_TARGET" == "deps" ]]; then
 
   exit 0
 elif [[ "$CI_TARGET" == "verify_examples" ]]; then
-  run_ci_verify "*" "wasm-cc|win32-front-proxy|shared"
+  run_ci_verify "*" "win32-front-proxy|shared"
```
Comment (adisuissa) [inline ci/do_ci.sh:508]: I may be missing some context, why is `wasm-cc` not being validated anymore?
it's still part of the examples, so should be validated, no?

**Review by adisuissa — COMMENTED** [review]
Thanks!

File: ci/do_ci.sh (hunk 1 above)
Comment (phlax) [reply ci/do_ci.sh:508]: in this line its not being _excluded_

it is being validated now in this test - just not separately

**Review by adisuissa — APPROVED** [review]
LGTM, thanks!

**Pull request merged**